NBitcoin/IpExtensions.cs

```diff
@@ -138,6 +138,36 @@ public static bool IsRFC4843(this IPAddress address)
 			return (bytes[15 - 15] == 0x20 && bytes[15 - 14] == 0x01 && bytes[15 - 13] == 0x00 && (bytes[15 - 12] & 0xF0) == 0x10);
 		}
 
+		public static byte[] GetGroup(this EndPoint endpoint)
+		{
+			if (endpoint is IPEndPoint ip)
+			{
+				return ip.Address.GetGroup();
+			}
+			else if (endpoint is DnsEndPoint dns)
+			{
+				var vchRet = new List<byte>();
+				if (dns.IsTor() || dns.IsI2P())
+				{
+					if (dns.IsTor())
+					{
+						vchRet.Add((byte)1);
+					}
+					else if (dns.IsI2P())

```

Comment:
IsCjdns condition missing from this if-else, is that intentional?

Reply:
CJDNS are IPv6 addresses.

Reply:
ACK

Reply:
`else if (dns.IsI2P())` seems superfluous because simple `else` would do the job anyway.